CONTRIBUTING.md

```diff
@@ -0,0 +1,232 @@
+<!-- omit in toc -->
+# Regarding contributions
+
+All types of contributions are encouraged and valued. See the [Table of contents](#table-of-contents)
+for different ways to help and details about how this project handles them. Please make sure to read
+the relevant section before making your contribution. It will make it a lot easier for us maintainers
+and smooth out the experience for all involved. We look forward to your contributions.
+
+> And if you like the project, but just don't have time to contribute, that's fine. There are other
+> easy ways to support the project and show your appreciation, which we would also be very happy about:
+>
+> - Star the project
+> - Tweet about it
+> - Refer this project in your project's readme
+> - Mention the project at local meetups and tell your friends/colleagues
+
+<!-- omit in toc -->
+## Table of contents
+
+- [I have a question](#i-have-a-question)
+- [I want to contribute](#i-want-to-contribute)
+  - [Reporting bugs](#reporting-bugs)
+    - [Before submitting a bug report](#before-submitting-a-bug-report)
+    - [How do I submit a good bug report?](#how-do-i-submit-a-good-bug-report)
+  - [Suggesting enhancements](#suggesting-enhancements)
+    - [Before Submitting an Enhancement](#before-submitting-an-enhancement)
+    - [How do I submit a good enhancement suggestion?](#how-do-i-submit-a-good-enhancement-suggestion)
+  - [Your first code contribution](#your-first-code-contribution)
+  - [Lint of codebase](#lint-of-codebase)
+    - [Linters in GitHub Action](#linters-in-github-action)
+    - [Run super-linter locally](#run-super-linter-locally)
+    - [Run super-linter interactively](#run-super-linter-interactively)
+
+## I have a question
+
+Before you ask a question, it is best to search for existing [issues][issues] that might help you.
+In case you have found a suitable issue and still need clarification, you can write your question in
+this issue. It is also advisable to search the internet for answers first.
```

Comment:
you can write your question in the 'issues' section - Could be more understandable?